app/daos/OrganizationDAO.scala

```diff
@@ -86,6 +99,13 @@ class MariadbOrganizationDAO extends OrganizationDAO {
       p.head.id }), 10 second )
     dbConfig.db.run((profileOrganizations.filter(uo => uo.organizationId === organization_id && uo.profileId === profile_id)).exists.result)
   }
+  def checkProfileOranization(profileEmail: String, organizationName:String): Future[Boolean] = {
```

Comment:
Same here as before: `Future` instead of `Await` and a `private` method that can be used?